bin/change_headers_to_links.sh

```diff
@@ -2,7 +2,7 @@
 
 #---------------------------------------------------------------------------
 # Little helper script to turn include header files into symbolic
-# links -- needed because we need to prefix the sources for ln -s [in 
+# links -- needed because we need to prefix the sources for ln -sf [in 
 # its $(LN_S) incarnation] with their full directory.
```

Comment:
Including `-f` for the soft linking command `ln -s` ensures that the link is created even if we are overwriting the file. This was causing rebuilding the library to fail when testing on my macbook.
Let me know if you want me to remove these forcing flags.